steps/datadog-mobile-app-upload/1.0.1/step.yml

```diff
@@ -0,0 +1,82 @@
+title: Datadog Mobile App Upload
+summary: |
+  Step to upload a new version of you application to Datadog for Synthetic tests
+description: |
+  Step to upload a new version of you application to Datadog for Synthetic tests
+
+  With this Step you can upload a new version of your application to Datadog to run Synthetic tests against during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle.
+
+  To use this Step:
+  - Create Synthetic mobile tests in Datadog. As part of this you'll be asked to add a mobile application. An existing mobile application is a prerequisite for this command. See [Continuous Testing and CI/CD Configuration documentation](https://docs.datadoghq.com/continuous_testing/cicd_integrations/configuration?tab=npm) at the bottom of the page for more information.
+  - Upload and build your app in the CI.
+  - Configure and run this Step in your Bitrise workflow to upload a new version of your application to Datadog to test against.
+
+  This Step will:
+  - Upload a new version of an already existing application to Datadog to be used for testing.
+
+  For examples of how the inputs of this Step can be configured check the [README for this step](https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application)
+website: https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application
+source_code_url: https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application
+support_url: https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application/issues
+published_at: 2024-04-18T18:09:20.584036+02:00
+source:
+  git: https://github.com/DataDog/synthetics-test-automation-bitrise-step-upload-application.git
+  commit: ea10574c8be4872209bf5d3a934d32da193749e2
+project_type_tags:
+- ios
+- android
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: ./upload-application.sh
+inputs:
+- api_key: $DATADOG_API_KEY
+  opts:
+    category: Required Inputs
+    description: The API key used to query the Datadog API.
+    is_expand: true
+    is_required: true
+    title: Datadog API Key
+- app_key: $DATADOG_APP_KEY
+  opts:
+    category: Required Inputs
+    description: The application key used to query the Datadog API.
+    is_expand: true
+    is_required: true
+    title: Datadog APP Key
+- config_path: ""
+  opts:
+    category: Optional Inputs
+    description: The global JSON configuration used when launching tests.
+    title: Global JSON configuration
+- latest: false
+  opts:
+    category: Optional Inputs
+    description: Marks the application as `latest`. Any tests that run on the latest
+      version will use this version on their next run.
+    title: Latest
+- mobile_application_id: ""
+  opts:
+    category: Optional Inputs
+    description: ID of the application you want to upload the new version to.
+    is_required: true
```

Comment:
Adding `is_required` for this field and the 2 bellow is not a breaking change as the upload-application command we use does not work without them present, so this just reflects closer the actual behaviour